rplugin/node/nvim_typescript/src/index.ts

```diff
@@ -178,6 +179,10 @@ export default class TSHost {
     }
   }
 
+  isRenameSuccess(obj: protocol.RenameInfoSuccess | protocol.RenameInfoFailure): obj is protocol.RenameInfoSuccess {
```

Comment:
Why add this? Does the inline check not work?

Reply:
yes it was failing to build for me